### ISSUE TITLE
Unable to login after logging out 

### ISSUE THREAD

**Issue by mponce1** [github/VisualStudio#277]
Steps to reproduce: 
1. Log into GHfVS via team explorer > manage connections > github
2. Log out
3. Login with the same user or any other user

Result: Unable to connect to server error message. 

Extension.log: 
```
2016-05-02 16:32:27.8143|INFO|thread: 1|LoginTabViewModel|Error logging into 'https://github.com/' as 'mergecity'
System.ObjectDisposedException: The cache 'SqlitePersistentBlobCache' was disposed.
```

![screen shot 2016-05-02 at 4 32 47 pm](https://cloud.githubusercontent.com/assets/6272408/14971075/850bb3cc-1083-11e6-80cf-9f33869cc79a.png)

**Comment by grokys** [comment]
Can't repro this on master. Was it fixed by #280?

**Comment by grokys** [comment]
Closing as this seems to be fixed.
